release_docs/INSTALL

```diff
@@ -70,8 +55,35 @@ CONTENTS
 
         https://github.com/HDFGroup/hdf5/releases
 
-4. Build and Install HDF5 on Unix and Mac OSX Platforms with Autotools
-        see the release_docs/INSTALL_Autotools.txt file.
-
-5. Build and Install HDF5 Libraries and Tools with CMake
+4. Build and Install HDF5 Libraries and Tools with CMake
         see the release_docs/INSTALL_CMake.txt file.
+
+5. Quick Start Presets
+        You want to build HDF5 with CMake, but there are so many options to consider.
+
+5.1. Solution
+        CMake introduced presets in version 3.19. HDF Group provides a file in the source,
+        CMakePresets.json, requiring CMake 3.24 or higher, that will build HDF5 with the
+        options for building a typical shared library with the common languages for a
+        platform. The features include building the tools, examples, plugins, and the
+        shared and static libraries.
+
+5.2. Discussion
+        The CMakePresets.json file is located in the root directory of the HDF5 source.
+        It is from here you will execute the cmake command to build HDF5. The following
+        example shows how to build HDF5 with the CMakePresets.json file:
+            change directory to the hdf5 source folder
+            execute "cmake --workflow --preset ci-StdShar-<compiler-type> --fresh"
+                where "<compiler-type>" is  GNUC or MSVC or Clang
+
+        The above example will create a "build" folder in the source parent directory, which
+        will contain the results of the build, including installation package files.
+
+5.3. Customization
+        See the HDF5 documentation for more on presets, especially the presets section in:
```

Comment:
If HDF5 documentation is different than the INSTALL_CMake.txt mentioned later, then a link to the specific spot will be helpful.